### ISSUE TITLE
amp-video integration test failure

### ISSUE THREAD

**Issue by muxin** [ampproject/amphtml#7290]
```
Safari 8.0.8 (Mac OS X 10.10.5) amp-video Actions "before all" hook FAILED
	TypeError: null is not an object (evaluating 'a.__AMP_TOP') (http://localhost:9876/dist/v0.js:44)
Safari 8.0.8 (Mac OS X 10.10.5) amp-video Actions "after each" hook: cleanUp for "should support mute, play, pause, unmute actions" FAILED
	undefined is not an object (evaluating 'fn.call')
Safari 8.0.8 (Mac OS X 10.10.5) amp-video Autoplay play/pause should play when in view port initially FAILED
	TypeError: undefined is not a function (evaluating 'this.win.setTimeout(a,b)') (http://localhost:9876/dist/v0.js:225)
.
Safari 8.0.8 (Mac OS X 10.10.5) amp-video Autoplay "after each" hook: cleanUp for "should play when in view port initially" FAILED
	NotFoundError: DOM Exception 8
	removeChild@[native code]
	cleanUp@/tmp/5a0aa446d682e1f6a589e0eab75cdf17.browserify:43129:41 <- /home/travis/build/ampproject/amphtml/test/integration/test-video-players-helper.js:253:41
Safari 8.0.8 (Mac OS X 10.10.5) amp-video Autoplay "after each" hook: cleanUp for "should play when in view port initially" FAILED
	NotFoundError: DOM Exception 8
	removeChild@[native code]
	cleanUp@/tmp/5a0aa446d682e1f6a589e0eab75cdf17.browserify:43129:41 <- /home/travis/build/ampproject/amphtml/test/integration/test-video-players-helper.js:253:41
```
Seems like `removeChild` in `cleanUp` is not called properly. Why do we have to clean up anyway?

**Comment by aghassemi** [comment]
@muxin There are several Safari 8 issues. For now we are doing https://github.com/ampproject/amphtml/pull/7276 to handle it.

**Comment by aghassemi** [comment]
Closed by #7276
